cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -168,6 +168,9 @@ spec:
 {{- end }}
         resources:
 {{ toYaml .Values.resources | indent 10 }}
+{{- if .Values.useHostNetwork }}
+      hostNetwork: true
```

Comment:
@ruslanloman The helm chart currently does not include the settings for the CephCluster CR (cluster.yaml). The helm chart effectively only includes the settings from common.yaml and operator.yaml. I'm not seeing how this change will help accomplish setting up host networking.

Reply:
Were you able to test this change? Something doesn't look right with the indentation, but maybe it's fine.

Reply:
In other places where rook configures host networking in the cluster we also set `dnsPolicy: ClusterFirstWithHostNet`. Do you need that here?